common/djangoapps/service_status/urls.py

```diff
@@ -1,15 +0,0 @@
-"""
```

Comment:
Reviewers: I'm not sure why GitHub sees this as a delete and an add. It was just moved along with the other files.